src/components/Album.js

```diff
@@ -10,9 +11,11 @@ function Album({ album }) {
         <p className="album-title">{album.albumName}</p>
         <p className="album-artist">{album.artistName}</p>
         <div className="button-container">
-          <Button variant="contained" color="primary" className="buy-button">
+          {/* <Button variant="contained" color="primary" className="buy-button">
             Buy
-          </Button>
+          </Button> */}
+          {/*AddToCartButton component */}
+          <AddToCartButton album={album} />{' '}
```

Comment:
Is the `{' '}` here for spacing purposes or something else?

Reply:
It's Prettier's important.

Reply:
ah okay if the formatter wants to put it there then that's fine, we'll live with it :sweat\_smile: